firebase-firestore/src/main/java/com/google/firebase/firestore/local/LruDelegate.java

```diff
@@ -49,4 +49,7 @@
 
   /** Access to the underlying LRU Garbage collector instance. */
   LruGarbageCollector getGarbageCollector();
+
+  /** Return the size of the cache in bytes. */
+  long getByteSize();
```

Comment:
`getByteSize()` doesn't really convey what this method measures. Maybe `getCacheSize()` or `getCacheSizeInBytes()`?